### ISSUE TITLE
(URGENT) Unable to create Change Request in Translate and Create tabs

### ISSUE THREAD

**Issue by ronaldtse** [glossarist/glossarist-desktop#108]
If I go to the Translate or Create tabs, the "+ New change request" button doesn't work.

This means I am unable to create any term or translate any term.

<img width="1312" alt="Screen Shot 2020-09-03 at 2 48 41 PM" src="https://user-images.githubusercontent.com/11865/92080725-92dd4800-edf4-11ea-9252-d4eca7baecd5.png">

**Comment by ronaldtse** [comment]
One workaround is possible, which is to "Edit" a term (which creates a CR with the green + button), to create the CR, and then do the Create or Translate. But this doesn't work at all for new glossary repositories because there are no terms to edit.

**Comment by strogonoff** [comment]
Right, creation screen is not supporting the auto-created CRs—will be fixed

**Comment by ronaldtse** [comment]
The "creation screen" is fixed, but not the "translate screen":
<img width="1312" alt="Screen Shot 2020-09-08 at 6 57 50 PM" src="https://user-images.githubusercontent.com/11865/92468155-4c795600-f205-11ea-9dbb-876aa3f0f6cb.png">

**Comment by ronaldtse** [comment]
With the latest Glossarist we still can't add a term!!

1. Open Glossarist:

<img width="1312" alt="Screen Shot 2020-09-09 at 8 07 19 AM" src="https://user-images.githubusercontent.com/11865/92539247-7c5f4280-f273-11ea-8412-a2488abf50cc.png">

2. Click Create:
<img width="1312" alt="Screen Shot 2020-09-09 at 8 07 30 AM" src="https://user-images.githubusercontent.com/11865/92539261-841ee700-f273-11ea-8f25-e2098368525c.png">

3. Click Proceed:
<img width="1312" alt="Screen Shot 2020-09-09 at 8 07 40 AM" src="https://user-images.githubusercontent.com/11865/92539266-897c3180-f273-11ea-97bc-d4d1b07efce4.png">

4. Enter something
<img width="1312" alt="Screen Shot 2020-09-09 at 8 08 01 AM" src="https://user-images.githubusercontent.com/11865/92539281-9567f380-f273-11ea-9f92-5974e111e438.png">

5. Click "Start CR"
<img width="1312" alt="Screen Shot 2020-09-09 at 8 08 09 AM" src="https://user-images.githubusercontent.com/11865/92539292-9ac53e00-f273-11ea-8504-71acf28e38e5.png">

6. CR created
<img width="1312" alt="Screen Shot 2020-09-09 at 8 08 22 AM" src="https://user-images.githubusercontent.com/11865/92539298-a284e280-f273-11ea-983d-92c338c4eaef.png">

7. Go to Review
<img width="1312" alt="Screen Shot 2020-09-09 at 8 08 33 AM" src="https://user-images.githubusercontent.com/11865/92539308-a87ac380-f273-11ea-83c5-ae0a1cb4987d.png">

8. Click on the new item
<img width="1312" alt="Screen Shot 2020-09-09 at 8 08 46 AM" src="https://user-images.githubusercontent.com/11865/92539316-b03a6800-f273-11ea-8daf-c76e172ca814.png">

9. Click "Propose"
<img width="1312" alt="Screen Shot 2020-09-09 at 8 09 01 AM" src="https://user-images.githubusercontent.com/11865/92539334-b92b3980-f273-11ea-8d51-9477e29835b2.png">

10. Change filter to "Submitted"
<img width="1200" alt="Screen Shot 2020-09-09 at 8 09 18 AM" src="https://user-images.githubusercontent.com/11865/92539348-c34d3800-f273-11ea-94b5-9522b79f0c7b.png">

11. See proposed change request
<img width="1312" alt="Screen Shot 2020-09-09 at 8 09 31 AM" src="https://user-images.githubusercontent.com/11865/92539355-cc3e0980-f273-11ea-9fc4-bc0a7524b77c.png">

12. Update ID to accept.
<img width="1312" alt="Screen Shot 2020-09-09 at 8 09 49 AM" src="https://user-images.githubusercontent.com/11865/92539376-d5c77180-f273-11ea-8472-dfc51119c114.png">

13. The ACCEPT REVISION button *IS DISABLED*!
<img width="1312" alt="Screen Shot 2020-09-09 at 8 09 59 AM" src="https://user-images.githubusercontent.com/11865/92539389-dd871600-f273-11ea-85ae-10ead15fd768.png">

**Comment by ronaldtse** [comment]
Here's the repo I'm using with: https://github.com/ronaldtse/my-glossary/blob/master/roles.yaml

**Comment by ronaldtse** [comment]
Worse --

1. I "Withdraw" the CR:
<img width="1312" alt="Screen Shot 2020-09-09 at 8 12 07 AM" src="https://user-images.githubusercontent.com/11865/92539480-28089280-f274-11ea-90a2-20f43655303d.png">

2. CR is Withdrawn
<img width="1312" alt="Screen Shot 2020-09-09 at 8 12 13 AM" src="https://user-images.githubusercontent.com/11865/92539488-2c34b000-f274-11ea-80d1-5451a26a8b44.png">

3. Go to Create again. Then I'm shown the same term even it was withdrawn, and I can't escape!
<img width="1312" alt="Screen Shot 2020-09-09 at 8 12 46 AM" src="https://user-images.githubusercontent.com/11865/92539510-3f478000-f274-11ea-8e6e-bfb5167a0114.png">

4. I click on "Remove from CR"

Then the "Remove from CR" button is stuck. (the term name is different because I was trying)
<img width="1312" alt="Screen Shot 2020-09-09 at 8 15 12 AM" src="https://user-images.githubusercontent.com/11865/92539653-ac5b1580-f274-11ea-9c73-60231d7a15b3.png">

**Comment by ronaldtse** [comment]
On this screen, when I click on "(new CR draft)"
<img width="1312" alt="Screen Shot 2020-09-09 at 8 13 30 AM" src="https://user-images.githubusercontent.com/11865/92539540-59815e00-f274-11ea-995a-cd3cd4706616.png">

The screen clears
<img width="1312" alt="Screen Shot 2020-09-09 at 8 17 38 AM" src="https://user-images.githubusercontent.com/11865/92539731-edebc080-f274-11ea-8a97-6a3018d5225c.png">

So it is inconsistent on what the "(new CR draft)" button does -- it's a mix of the old and new behavior.

**Comment by strogonoff** [comment]
NB: There are a bunch of different issues in this thread that are hard to follow… They do not concern the inability to use create/translate modules that was fixed in 1.6.27–1.6.28.

> The ACCEPT REVISION button IS DISABLED!

Need to specify roles. Only register manager can accept revisions.

>  Go to Create again. Then I'm shown the same term even it was withdrawn, and I can't escape!

The selected item is global. If you change a module, item does not get reset. Changing CR state marker to “Withdrawn” leaves CR and current concept selected until you select something else.

> Then the "Remove from CR" button is stuck. (the term name is different because I was trying)

This one I’ll need to look into, couldn’t easily reproduce.

cc @ronaldtse

**Comment by strogonoff** [comment]
> The selected item is global. If you change a module, item does not get reset. Changing CR state marker to “Withdrawn” leaves CR and current concept selected until you select something else.

Filed https://github.com/glossarist/glossarist-desktop/issues/134

**Comment by ronaldtse** [comment]
This is all fixed, closing.
